opsdroid/core.py

```diff
@@ -254,28 +254,9 @@ def setup_skills(self, skills):
                     continue
 
                 if hasattr(func, "skill"):
-                    _LOGGER.warning(
-                        _(
-                            "Function based skills are deprecated "
-                            "and will be removed in a future "
-                            "release. Please use class-based skills "
-                            "instead."
-                        )
-                    )
                     func.config = skill["config"]
                     self.skills.append(func)
 
-        with contextlib.suppress(AttributeError):
-            for skill in skills:
-                skill["module"].setup(self, self.config)
```

Comment:
I showed you this line to check the warning raised, but we still need to setup each module. 

So all you need to do is remove the `_LOGGER.warning` but keep all the rest.

Reply:
Thanks Fabio, my bad, i will fix it and push it.